### ISSUE TITLE
Use without SRT file

### ISSUE THREAD

**Issue by mnlbox** [ahgsql/remotion-subtitles#1]
Hi @ahgsql ,

First thanks for this awesome project. It's a very nice idea.
Just a quick question:
Can we use these captions without a SRT file.
For example I want to use this Caption component with a normal text for the Intro of my video. My Intro is a normal Remotion sequence component. Something like this:

```
<Series>
     <Series.Sequence durationInFrames={85}>
          < TypewriterCaption text="Welcome to my video" durationInFrames={30} />
          {/* My other components in the Sequence here */}
     </Series.Sequence>
<Series>
```

**Comment by ahgsql** [comment]
Yeah of course, you need to handling timings by yourself and you are good to go

**Comment by mnlbox** [comment]
How can I handle timing in `Caption` level? Is there any property like `durationInFrames` for this component or do I need to wrap the component in another `<Series.Sequence>` in a nested way?
